src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/WageTypeMapper.cs

```diff
@@ -5,12 +5,15 @@ namespace Esfa.Vacancy.Application.Commands.CreateApprenticeship
 {
     public class WageTypeMapper : IWageTypeMapper
     {
-        public LegacyWageType MapToLegacy(WageType originalWageType)
+        public LegacyWageType MapToLegacy(CreateApprenticeshipRequest request)
```

Comment:
I don't like long method signatures, but in this case I think passing the whole request object is slightly overkill, perhaps just add Min and Max along side type.